package.json

```diff
@@ -33,7 +33,7 @@
   ],
   "main": "dist/index.cjs.js",
   "module": "dist/index.esm.js",
-  "files": ["dist"],
+  "files": ["src", "dist"],
```

Comment:
This would bloat the size of every NPM install, and not provide any value other than exposing the types.

Reply:
I'm not completely happy with this either but this is the "supported" solution currently. I think exposing types is a big value for users though.